server/src/test/java/com/google/fhir/proxy/BearerAuthorizationInterceptorTest.java

```diff
@@ -112,6 +117,24 @@ private String generateKeyPairAndEncode() {
     return null;
   }
 
+  private BearerAuthorizationInterceptor createTestInstance(boolean returnedCheck)
```

Comment:
nit: the `returnedCheck` var name is not clear as to what it represents. How about renaming to `isAccessGranted` or something similar?

Reply:
Done